xml/Menu/iats.xml

```diff
@@ -12,8 +12,8 @@
     <access_arguments>make online contributions</access_arguments>
   </item>
   <item>
-    <path>civicrm/admin/contribute/iatssettings</path>
-    <page_callback>CRM_Iats_Form_Settings</page_callback>
+    <path>civicrm/admin/setting/iats</path>
+    <page_callback>CRM_Admin_Form_Generic</page_callback>
```

Comment:
As per earlier note - does this also put the new path in the navigation menu? I think it does. Any reason we can't leave it where it is, i.e. the old path, and then we won't move or break the existing navMenu entry? I prefer the settings to appear within the contribution configuration, it makes more sense to me there than in the generic settings with all the other system configs.

Reply:
@adixon in regards to the url the Generic form assumes the last part of the url is the extension key https://github.com/civicrm/civicrm-core/blob/master/CRM/Admin/Form/SettingTrait.php#L142 I suppose that we could just do admin/contribute/iats but not sure if that woks and the format of the url is the same as in the documentation https://docs.civicrm.org/dev/en/latest/framework/setting/extension/